File: 1633-minimum-number-of-increments-on-subarrays-to-form-a-target-array/1633-minimum-number-of-increments-on-subarrays-to-form-a-target-array.go
func minNumberOperations(target []int) int {
	ans := target[0]
	for i := 1; i < len(target); i++ {
		diff := target[i] - target[i-1]
		if diff > 0 {
			ans += diff
		}
	}
	return ans
}

File: 1633-minimum-number-of-increments-on-subarrays-to-form-a-target-array/1633-minimum-number-of-increments-on-subarrays-to-form-a-target-array.rs
impl Solution {
    pub fn min_number_operations(target: Vec<i32>) -> i32 {
        let mut ans = target[0];
        for i in 1..target.len() {
            let diff = target[i] - target[i - 1];
            if diff > 0 {
                ans += diff;
            }
        }
        ans
    }
}